### PR TITLE
Export resources as OSGI

### PR DESCRIPTION
**Pull request by tdkmalan**
According to the Smooks known issue with OSGI [1] ,  ../META-INF/MANIFEST.MF.  should contain the resource META-INF.xsd.smooks. 

With this fix,  above resources are export to OSGI.

[1] https://github.com/smooks/smooks/issues/94

## Purpose
https://github.com/smooks/smooks/issues/94

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: smooks/1.5.1.wso2v3/pom.xml [hunk 1]
```diff
@@ -0,0 +1,228 @@
+<!--
+~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thusithathilina) [inline smooks/1.5.1.wso2v3/pom.xml]: Year should be 2018

File: smooks/1.5.1.wso2v3/pom.xml [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+                    <artifactId>ognl</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>javassist</groupId>
+                    <artifactId>javassist</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+    <repositories>
```
Comment (thusithathilina) [inline smooks/1.5.1.wso2v3/pom.xml]: Why do you need wso2 repo?

File: smooks/1.5.1.wso2v3/pom.xml [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+            </releases>
+        </repository>
+    </repositories>
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+        <snapshotRepository>
```
Comment (thusithathilina) [inline smooks/1.5.1.wso2v3/pom.xml]: Remove snapshot distro

File: smooks/1.5.1.wso2v3/pom.xml [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+                            org.zeroturnaround.javarebel;resolution:=optional,
+                            sun.misc;resolution:=optional
+                        </Import-Package>
+                        <Private-Package>
+                        </Private-Package>
+                        <Include-Resource>
+                            @milyn-smooks-all-${org.milyn.version}.jar!/META-INF/*,
+                            @milyn-smooks-all-${org.milyn.version}.jar!/smooks-regex.properties
+                        </Include-Resource>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```
Comment (thusithathilina) [inline smooks/1.5.1.wso2v3/pom.xml]: Does this require DynamicImports?

File: smooks/1.5.1.wso2v3/pom.xml [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+                            org.yaml.*;version="${org.yaml.version}",
+                            org.mvel2.*;version="${org.mvel2.version}"
+                        </Private-Package>
+                        <Export-Package>
+                            META-INF.xsd.smooks,
+                            META-INF.xsd.smooks.*;-split-package:=merge-first,
+                            org.milyn.*;version="${org.milyn.osgi.version}"
+                        </Export-Package>
+                        <Import-Package>
+                            au.com.bytecode.opencsv;resolution:=optional,
```
Comment (Kishanthan) [inline smooks/1.5.1.wso2v3/pom.xml]: Most of the import packages does not have version (range). The best practice is to use proper import versions.

**Comment by Kishanthan** [comment]
This PR needs improvement mentioned in the review comment, hence closing this. Please reopen once the improvement is added.

**Pull request closed without merging**